frontend/src/scenes/persons/PersonsTable.tsx

```diff
@@ -34,14 +37,20 @@ export function PersonsTable({
     loadPrevious,
     loadNext,
     allColumns,
-    cohort,
+    backTo = 'Persons',
+    sessionsFilters = [],
+    date = undefined,
 }: PersonsTableType): JSX.Element {
-    const linkToPerson = (person: PersonType): string => {
-        const backTo = cohort
-            ? `#backTo=Cohorts&backToURL=${window.location.pathname}`
-            : `#backTo=Persons&backToURL=${window.location.pathname}`
-        return `/person/${encodeURIComponent(person.distinct_ids[0])}${backTo}`
-    }
+    const deepLinkToPersonSessions = (person: PersonType): string =>
+        combineUrl(
+            `/person/${encodeURIComponent(person.distinct_ids[0])}`,
+            { filters: sessionsFilters, date },
+            {
+                backTo,
+                backToURL: window.location.pathname + window.location.search + window.location.hash,
+                activeTab: backTo === 'Insights' ? PersonsTabType.SESSIONS : PersonsTabType.EVENTS,
+            }
+        ).url
```

Comment:
For later, but relevant: we should gather all different ways we can link between pages and all the extension logic into some utils folder. This could definitely go there.

I did some early work on this, but from a different angle here: https://github.com/PostHog/posthog/pull/4656 -- these two could be merged somehow?

Reply:
Interesting thats a good point. Yea this definitely felt hacky but my changes is a slight upgrade from what we had originally. It'd be nice to bring this under your sceneLogic once your PR is merged.